### ISSUE TITLE
Add `Signature` entity

### ISSUE THREAD

**Issue by seandavi** [seandavi/MungeCuratedMGS#1]

**Comment by lgeistlinger** [comment]
I think we are still missing this one though.

Also, I think:

1. the `signature` table should have a column `metadata` that contains the corresponding key of the `signature_metadata` table. 

2. the `signature_metadata` table should have a column `study` that contains the corresponding key of the `study` table.

3. Following 1 and 2, we can remove the `location` and `citation` columns from the `signature_metadata` table, as this is annotated in the `study` table.

**Comment by lgeistlinger** [comment]
According to the data model, the `signature` table should have a column for each alpha-diversity measure and also store the signature as composed of the various taxa.

Should the actual signature be represented as one column (storing the taxa eg. comma-separated). Or should the signature, as currently in the spreadsheet, be represented by an open-ended number of columns - with one taxon per column?

**Comment by lwaldron** [comment]
In the database, the signature will be a table with (likely) two columns: 1) taxonomic nomenclature, and 2) NCBI ID (where available). I guess "up/down/not changed/present" goes with the signature metadata table.

Actually I think the alpha diversity is conceptually a separate table, with only one column for the name of the alpha diversity measure. I specified "not changed" above for any alpha diversity measures calculated but that weren't changed, as opposed to those not measured. One then *could* also create a signature of microbial taxa not changed, although obviously it wouldn't be required. I added "present" above to allow for signatures that are about presence rather than differential abundance. We may have to change the signature metadata, or make another different metadata table, once we start entering physiology / morphology signatures.
